liiatools/spec/cin_census/cin-2022.xsd

```diff
@@ -194,6 +201,7 @@
 
   <xs:simpleType name="gendertype">
     <xs:restriction base="xs:string">
```

Comment:
@patrick-troy I think you could replace base="xs:string" with base="nonEmptyString" to be a bit more DRY

Reply:
Ah yes very nice, have done this now